modules/oe_bootstrap_theme_helper/tests/src/Kernel/TwigExtensionTest.php

```diff
@@ -427,4 +428,143 @@ protected function elementChildrenFilterDataProvider(): array {
     return $scenarios;
   }
 
+  /**
+   * Tests BCL card list filter.
+   */
+  public function testBclCardList(): void {
+    $extension = $this->container->get('oe_bootstrap_theme_helper.twig_extension');
+    $data = $this->bclCardListDataProvider();
```

Comment:
Is there a reason to avoid the `@dataProvider` annotation?

Reply:
We tend to avoid data providers for tests from kernel and up, because it takes a huge time just to make a new install and the tested data is always so small.